src/containers/ftp/sftp_files.rb

```diff
@@ -19,9 +19,10 @@ def upload_file(uri, local_file, options = {})
     remote_path = CGI.unescape(uri.path) if uri.path
     remote_user = CGI.unescape(uri.user) if uri.user
     remote_password = CGI.unescape(uri.password) if uri.password
+    remote_port = uri.port || 22
     md5 = options[:md5].nil? ? false : options[:md5]
 
-    Net::SFTP.start(remote_host, remote_user, password: remote_password) do |sftp|
+    Net::SFTP.start(remote_host, remote_user, password: remote_password, port: remote_port) do |sftp|
```

Comment:
yeah, looks like it calls ssh start under the covers
https://www.rubydoc.info/github/net-ssh/net-ssh/Net/SSH#start-class_method
and that accepts a `:port` option
`:port => the port to use when connecting to the remote host`